LINAC.c

```diff
@@ -0,0 +1,33 @@
+#include <xc.h>
+#include "LINAC.h"
+
+ void LINAC_init() {
+    TRISB1 = 0;     //set MOTOR_FWD as output
+    LATB1 = 0;
+    
+    TRISB2 = 0;     //set MOTOR_BWD as output
+    LATB2 = 0;
+    
+    TRISB3 = 1;     //set LIM_HIGH as input
+    ANSELB3 = 0;    //set as digital input
+    
+    TRISB4 = 1;     //set LIM_LOW as input
+    ANSELB4 = 0;    //set as digital input
+    
+}
+ 
+int open_vent(){
+    LATB1 = 1;      //FWD High
+    LATB2 = 0;      //BWD Low
+    while(RB3)
```

Comment:
This function should not block. It should just set the relevant pins and exit. This is so that the main event loop runs as quickly as possible. More seriously, if the valve gets stuck, the entire board is stuck in an infinite loop. Valve status will be checked independently of this function.